src/appkit.cs

```diff
@@ -25201,4 +25762,282 @@ interface NSSliderAccessoryBehavior : NSCoding, NSCopying
+	{
+		[Abstract]
+		[Export ("rotor:resultForSearchParameters:")]
+		[return: NullAllowed]
+		NSAccessibilityCustomRotorItemResult ResultForSearch (NSAccessibilityCustomRotor rotor, NSAccessibilityCustomRotorSearchParameters searchParameters);
+	}
+
+	interface INSAccessibilityElementLoading {}
+
+	[Mac (10,13)]
+	[Protocol, Model]
+	[BaseType (typeof(NSObject))]
+	interface NSAccessibilityElementLoading
+	{
+		[Abstract]
+		[Export ("accessibilityElementWithToken:")]
+		[return: NullAllowed]
+		NSAccessibilityElement GetAccessibilityElement (INSSecureCoding token);
+
+		[Export ("accessibilityRangeInTargetElementWithToken:")]
+		NSRange GetAccessibilityRangeInTargetElement (INSSecureCoding token);
+	}
+
+	interface INSCollectionViewPrefetching { }
+
+	[Mac (10,13)]
+	[Protocol, Model]
+	[BaseType (typeof(NSObject))]
+	interface NSCollectionViewPrefetching
+	{
+		[Abstract]
+		[Export ("collectionView:prefetchItemsAtIndexPaths:")]
+		void PrefetchItems (NSCollectionView collectionView, NSIndexPath[] indexPaths);
+
+		[Export ("collectionView:cancelPrefetchingForItemsAtIndexPaths:")]
+		void CancelPrefetchingForItems (NSCollectionView collectionView, NSIndexPath[] indexPaths);
+	}
+
+	delegate bool DownloadFontAssetsRequestCompletionHandler (NSError error);
+
+	[Mac (10,13)]
+	[BaseType (typeof(NSObject))]
+	[DisableDefaultCtor]
+	interface NSFontAssetRequest : INSProgressReporting
+	{
+		[Export ("initWithFontDescriptors:options:")]
+		[DesignatedInitializer]
+		IntPtr Constructor (NSFontDescriptor[] fontDescriptors, NSFontAssetRequestOptions options);
+
+		[Export ("downloadedFontDescriptors", ArgumentSemantic.Copy)]
+		NSFontDescriptor[] DownloadedFontDescriptors { get; }
+
+		[Export ("progress", ArgumentSemantic.Strong)]
+		NSProgress Progress { get; }
+
+		[Export ("downloadFontAssetsWithCompletionHandler:")]
+		void DownloadFontAssets (DownloadFontAssetsRequestCompletionHandler completionHandler);
+	}
+
+	[Category]
+	[BaseType (typeof(NSObject))]
+	interface NSObject_NSFontPanelValidationAdditions
+	{
+		[Export ("validModesForFontPanel:")]
+		NSFontPanelModeMask ValidModesForFontPanel (NSFontPanel fontPanel);
+	}
+
+	[Mac (10, 13)]
+	[BaseType (typeof(NSObject))]
+	interface NSUserInterfaceCompressionOptions : NSCopying, NSCoding
+	{
+		[Export ("initWithIdentifier:")]
+		[DesignatedInitializer]
+		IntPtr Constructor (string identifier);
+
+		[Export ("initWithCompressionOptions:")]
+		[DesignatedInitializer]
+		IntPtr Constructor (NSSet<NSUserInterfaceCompressionOptions> options);
+
+		[Export ("containsOptions:")]
+		bool ContainsOptions (NSUserInterfaceCompressionOptions options);
+
+		[Export ("intersectsOptions:")]
+		bool IntersectsOptions (NSUserInterfaceCompressionOptions options);
+
+		[Export ("empty")]
+		bool Empty { [Bind ("isEmpty")] get; }
+
+		[Export ("optionsByAddingOptions:")]
+		NSUserInterfaceCompressionOptions CreateOptionsByAdding (NSUserInterfaceCompressionOptions options);
```

Comment:
Actually since we return `NSUserInterfaceCompressionOptions` this should be `GetOptionsByAdding`.

We kinda need `ByAdding` because there's a `ByRemoving` method.

However, maybe we could introduce an enum there and do something like:

```
NSUserInterfaceCompressionOptions GetOptions (NSUserInterfaceCompressionOptions options, NSUserInterfaceCompressionOptionsMode mode);
```

Where `NSUserInterfaceCompressionOptions` has 2 values: `add` and `remove`.

Maybe we'd need to add API doc for that (: